src/docs/product/performance/event-detail.mdx

```diff
@@ -24,14 +24,28 @@ To find these views, you can either go through the [Transaction Summary](/produc
 
 #### With Transaction Summary
 
-Select the [Performance Homepage](/product/performance/), then click the affected transaction to display the trace data.
+Select the [Performance Homepage](/product/performance/), then click the affected transaction to display the transaction data.
 
-#### With Query Builder
+#### Querying for this Transaction
 
 Scroll down to the "Trace Details" context panel in either the Issue Details or the Discover Event Details page, and click on the "View Summary" button. This will maintain the context of the current Sentry event.
 
 _Note_: Users on the Team or Business plans can also view a list of transaction events by clicking on the "Transactions" pre-built query in [Discover](/product/discover-queries/) or by performing a search with the `event.type:transaction` condition the Discover Query Builder view.
 
+### Quick Trace
+
+The quick trace (which displays top right of the span view) is an abbreviated view of the related [trace](/product/performance/distributed-tracing) for the current transaction.
+The quick trace displays up to six nodes, each representing different groups of the event's trace:
+- **Root**: Starting transaction
+- **Ancestor**: Events that are descendants of the root, and are directly related to the current event
```

Comment:
Nit: using the term descendants to explain ancestors seems a little ironic.

Reply:
Heh, what about
```suggestion
- **Ancestor**: Events in the generations between the Root and the Parent of the current Event
```